packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:math';
+import 'dart:ui';
```

Comment:
nit: by convention typically `import 'dart:ui' as ui` as it has some names that can overlap with other libraries we use.

Reply:
It turns out this line wasn't necessary.